src/hyperion/external_interaction/callbacks/rotation/ispyb_callback.py

```diff
@@ -120,29 +126,37 @@ def activity_gated_start(self, doc: RunStart):
             self.uid_to_finalize_on = doc.get("uid")
         return super().activity_gated_start(doc)
 
-    def update_deposition(self, params):
-        dcg_info = populate_data_collection_group(
-            self.ispyb.experiment_type,
-            params.hyperion_params.detector_params,
+    def populate_info_for_update(
```

Comment:
Should: Why are we putting the info from the start doc back into this object? Happy to discuss in person